sdk/dotnet/webrtc/api/data_channel_interface.cs

```diff
@@ -1,47 +1,116 @@
-/*
```

Comment:
I wonder what happened to this file. Maybe you changed line endings by accident?